tests/test_storage.py

```diff
@@ -64,10 +64,6 @@ def test_file_save(data, **kwargs):
         assert "md5:{0}".format(m.hexdigest()) == checksum
 
         assert size == len(data)
-        assert loc == join(
```

Comment:
Now there is a dedicated test checking if the paths are generated correctly. I think it is not good to keep this checking here because, moreover, it should be adapted to use the new path creations, and that requires the develop of the same method which generates the paths here again... If you hardcode the expected path for testing here, it will crash if the user modify the configuration to something else.